### DISCUSSION TITLE
Database Error when saving new user

### DISCUSSION THREAD

**Discussion by adamrusted** [supabase/supabase#7847] [Questions]
Hi team,

I'm currently getting a 500 error 'Database error when saving new user' when trying to sign a user up via email and password. Apart from the change in auth provider, I've copied everything else from [this tutorial](https://egghead.io/lessons/supabase-use-supabase-to-subscribe-to-database-events-with-postgres-triggers) but seem to be getting the error. Other users there are commenting saying they've been having similar issues for the last few months - wondering if there's a fix out there?

Attached screenshots for reference...
![thumbnail_image](https://user-images.githubusercontent.com/1830867/180442023-853f5b28-5abe-4caa-a8d2-f058239bbcdb.png)
![thumbnail_image (1)](https://user-images.githubusercontent.com/1830867/180442027-5322a5fd-7a64-47e8-a1b4-3534f0d73c4f.png)

TIA

Adam

**Comment by GaryAustin1** [comment]
Without looking at the tutorial this is almost always the result of a insert trigger function on auth.users table.
You could look in the dashboard database/postgres logs for a hint as to the error and/or turn off the trigger (assuming you do have one) to see if that is the issue.
